docs/plans/20260514-codex-first-class-mode.md

```
@@ -0,0 +1,465 @@
+type opts struct {
+    // ... existing ...
+    Codex        bool `long:"codex" description:"use codex CLI as the executor for task, review, and finalize phases (skips external review)"`
+    PassClaudeMd bool `long:"pass-claude-md" description:"pass project CLAUDE.md to codex via project_doc_fallback_filenames; user-level ~/.claude/CLAUDE.md is NOT auto-passed but a one-time setup hint is shown (codex executor only)"`
+}
+```
+
+`--codex` sets `cfg.Executor = "codex"` and `cfg.ExternalReviewTool = "none"` (unless user explicitly passed `--external-review-tool` to override, in which case fail with a clear error message). `--pass-claude-md` without a codex executor (`--codex` or `executor = codex`) fails validation with a clear error.
+
+**Codex `-c` flag args** (built per invocation by `codexConfigOpts.cliArgs()`):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)